sortPeople.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"sort"
+	"time"
+)
+
+type Person struct {
+	firstName string
+	lastName  string
+	birthDay  time.Time
+}
+
+func (p Person) String() string {
+	return fmt.Sprintf("{%s %s %s}", p.firstName, p.lastName, p.birthDay.Format("2006-01-02"))
+}
+
+type People []Person
+
+func (p People) Len() int {
+	if len(p) <= 0 {
```

Comment:
What the reason to exit with error code there, I think it pretty normal to return zero length